pkg/utils/ocm.go

```diff
@@ -488,6 +516,22 @@ func IsManagementCluster(clusterID string) (isMC bool, err error) {
 	return false, nil
 }
 
+func IsHostedCluster(clusterID string) (bool, error) {
```

Comment:
If we are adding this function, wondering if would make sense to update other places where such checks is performed? 
Other wise it would be a lot of divergent stuff going around